src/incubator/WheelPicker/index.tsx

```diff
@@ -153,7 +169,12 @@ const WheelPicker = React.memo(
               height: Spacings.s9,
               borderColor: Colors.grey60
             }}
-          />
+            center
+          >
+            <Text marginL-80 text80M {...rightLabelProps} color={activeTextColor} style={rightLabelStyle}>
```

Comment:
Maybe we should change the function name / move the label out, since now it's not rendering only the `separators`